tests/test_appfs.py

```diff
@@ -1,26 +1,29 @@
 from __future__ import unicode_literals
 
-import unittest
-
+import pytest
 import six
 
-from fs.appfs import UserDataFS
-
-
-class TestAppFS(unittest.TestCase):
-    """Test Application FS."""
-
-    def test_user_data(self):
-        """Test UserDataFS."""
-        user_data_fs = UserDataFS("fstest", "willmcgugan", "1.0")
-        if six.PY2:
-            self.assertEqual(
-                repr(user_data_fs),
-                "UserDataFS(u'fstest', author=u'willmcgugan', version=u'1.0')",
-            )
-        else:
-            self.assertEqual(
-                repr(user_data_fs),
-                "UserDataFS('fstest', author='willmcgugan', version='1.0')",
-            )
-        self.assertEqual(str(user_data_fs), "<userdatafs 'fstest'>")
+from fs import appfs
+
+try:
+    from unittest import mock
+except ImportError:
+    import mock
```

Comment:
I wondered where `mock` is used.

Reply:
here is the search result where `mock` is really used.

https://github.com/PyFilesystem/pyfilesystem2/search?q=mock&unscoped_q=mock

Reply:
Oooh good point -- I took out the last usage and forgot to remove the import too.